### DISCUSSION TITLE
How to Make dot net dll

### DISCUSSION THREAD

**Discussion by pekovbg** [IndySockets/Indy#601] [Q&A]
Hello. I using older Indy.Sockets.dll version : 10.1.5.33678 but on new linux distribution application crashed . How i can build latest from git , to Dot Net framework dll.

**Comment by rlebeau** [comment]
Unfortunately, I can't really answer that, as I have never used Indy in .NET, so I don't know the required steps needed for that compilation.  I would imagine it would require compiling Indy using Delphi.NET, which is a long-dead product.

In any case, I would suggest you consider migrating your code to other native .NET solutions, because Indy's code hasn't been maintained/recompiled for .NET in a very long time, and Indy will be dropping all support for .NET in Indy 11 anyway.
